guardian/manage-frontend
# Removing Google Analytics

## What does this change?
We want to stop using Google Analytics (GA) to record activity on MMA. This PR removes the use of Google Ananlytics within the manage frontend repo.



## How to test
We will need to merge this change and ask someone with access to the Google Analytics dashboard to check that calls to GA have stopped.


## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
